packages/dds/merge-tree/src/test/client.getPosition.spec.ts

```diff
@@ -57,6 +57,23 @@ describe("client.getPosition", () => {
         assert.strictEqual(pos, -1);
     });
 
+    it("All Segments Removed", () => {
```

Comment:
I don't think this test needs to be here anymore - I originally was going to put all of the tests for the zamboni logic in this file, but that was before I created a separate file for those tests. I have a test for the same case in zamboni.spec.ts.